File: flink-test/src/main/scala/com/atguigu/bean/UserBehavior.scala
package com.atguigu.bean

/**
  * @ClassName UserBehavior
  * @Description TODO
  * @Author Xing
  * @Date 2021/4/13 19:58
  * @Version 1.0
  */
case class UserBehavior(userId:Long,
                        itemId:Long,
                        categoryId:Int,
                        behavior:String,
                        timestamp:Long) {
}

File: flink-core/src/main/java/com/atguigu/apitest/beans/UserBehavior.java
package com.atguigu.apitest.beans;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @ClassName UserBehavior
 * @Description TODO
 * @Author Xing
 * @Date 2021/4/13 14:07
 * @Version 1.0
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserBehavior {
    private Long userId;
    private Long itemId;
    private Integer categoryId;
    private String behavior;
    private Long timestamp;
}
